docs/tutorials/hyp3_insar_stack_for_ts_analysis.ipynb

```diff
@@ -357,10 +357,21 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 52,
    "id": "9c3cf17d",
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "982"
+      ]
+     },
+     "execution_count": 52,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
```

Comment:
Our existing notebook doesn't include any output cells; I think the notebook is more consistent/easier to use if we remove the output cells this PR adds in sections 3.2 and 3.3.